Framework/PythonInterface/core/inc/MantidPythonInterface/core/PythonStdoutChannel.h

```diff
@@ -27,4 +37,17 @@ class MANTID_PYTHONINTERFACE_CORE_DLL PythonStdoutChannel : public ConsoleChanne
   /// Constructor for PythonStdoutChannel
   PythonStdoutChannel();
 };
+
+/* /// Store in case class PythonStdoutChannel is not up to the task and needs to be replaced with this class
+class MANTID_PYTHONINTERFACE_CORE_DLL PyBindStdoutChannel : public StdoutChannel {
+
+public:
+  /// Constructor for PythonStdoutChannel
+  PyBindStdoutChannel() : StdoutChannel() {}
+
+private:
+  pybind11::scoped_ostream_redirect m_redirect;
```

Comment:
If the compiler is C++17 or later, will this `pybind11` work still? 

This is the list of compilers that it supports now.
```
Supported compilers
Clang/LLVM 3.3 or newer (for Apple Xcode’s clang, this is 5.0.0 or newer)
GCC 4.8 or newer
Microsoft Visual Studio 2015 Update 3 or newer
Intel classic C++ compiler 18 or newer (ICC 20.2 tested in CI)
Cygwin/GCC (previously tested on 2.5.1)
NVCC (CUDA 11.0 tested in CI)
NVIDIA PGI (20.9 tested in CI)
```

Reply:
But it seems that you "deleted" it already.

Reply:
I added a "TODO" line stating we should replace the current python channel to the one using pybind11, once this library is adopted in Mantid.